src/Capture.php

```diff
@@ -235,12 +242,16 @@ public function save($imageLocation, $deleteFileIfExists = true)
             $data['includedJsSnippets'] = $this->includedJsSnippets;
         }
 
+	    if ($this->blockAnalytics) {
+		    $data['blockAnalytics'] = $this->blockAnalytics;
+	    }
+
         if ($deleteFileIfExists && file_exists($this->imageLocation) && is_writable($this->imageLocation)) {
             unlink($this->imageLocation);
         }
 
         $jobName = md5(json_encode($data));
-        $jobPath = $this->jobs->getLocation() . $jobName . '.js';
+        $jobPath = $this->jobs->getLocation() . rand(1, 1000) . '.js';
```

Comment:
This is incorrect - I put that in there for testing only and failed to remove before pushing.